### DISCUSSION TITLE
Tranlsations and countries/cities list translations

### DISCUSSION THREAD

**Discussion by MattiaMarchiorato** [fattureincloud/api#222] [Q&A] [answered]
Hello,

Is there a way to retreive countries cities etc.
Using GET /info/countries we receive a list in IT, what about if we need it in EN?

Another problem is when we issue a document, we have to fill in https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents
country: string
Country ⤵

Default: Italia
country_iso: string
Country Iso

In php sdk we have only country method not country_iso method (not sure we can use country_iso instead country).
And the problem is that we have to use IT  state names when our platform is multi language.

Thanks

MM

**Comment by emajo** [comment, marked as answer]
Hi @Mattia-Marchiorato, yes you can use the `entity.country_iso` field, to get the right iso we have the [/info/detailed_countries](https://developers.fattureincloud.it/api-reference#get-/info/detailed_countries) method.
Probably the reason beacause you are not finding the country_iso field is that you are using an old version of the SDK, in fact it was added only recently, I suggest to update to the latest release and check if you can find it.

Thanks,
Emanuele.

**Comment by devInsology** [comment]
Ok, i updated the sdk and noi i can see the method.

Thanks

MM

**Comment by MattiaMarchiorato** [comment]
Hello, 

we have a problem, {"name":"Isole Canarie (Spagna)","settings_name":"Canary Islands","iso":"ES","fiscal_iso":"ES","uic":"100"}
{"name":"Spagna","settings_name":"Spain","iso":"ES","fiscal_iso":"ES","uic":"067"}
Hanno lo stesso iso code, se passiamo quindi country_iso  all'endpount quale ci collega? Uno a caso?

Abbiamo le api su un servizio in produzione, ora tutti i clienti spagnoli si sono magicamente trasferiti alle Canarie.

Vi prego datemi una spiegazione plausibile.

**Reply by valmoz** [reply]
Ciao @Mattia-Marchiorato 
abbiamo fornito una risposta sulla issue che ha aperto a riguardo: #243 

La prossima volta le chiederei gentilmente di non scrivere sotto a un post in cui è già stata selezionata una risposta, in quanto rende di difficile individuazione la richiesta di supporto (teniamo monitorate solo le richieste senza risposta) e in ogni caso di selezionare solo una tra Discussions e Issue, selezionando quella più adatta alla casistica (in questo caso, una Issue).

In caso di dubbio, è possibile aprire una discussion e convertirla poi in issue nel caso in cui venisse effettivamente riscontrato un problema.

Non proseguiremo la discussione su questo tema qui. Consigliamo di tener controllata la issue, che verrà aggiornata quando ci saranno novità a riguardo.

Grazie
Mauro
